### PR TITLE
Hookup services sync buttons to API

### PR DESCRIPTION
**Pull request by coverbeck**
dockstore/dockstore#2698

### CODE REVIEW

File: src/app/shared/refresh.service.ts [hunk 1]
```diff
@@ -139,4 +139,28 @@ export class RefreshService {
...
+      },
+      error => this.alertService.detailedError(error)
+    );
+  }
+
+  syncServicesForOrganziation(organization: string): void {
+    const message = 'Syncing services for organization ' + organization;
+    this.alertService.start(message);
+    this.usersService.syncUserServicesbyOrganization(organization).subscribe(
+      response => {
```
Comment (garyluu) [inline src/app/shared/refresh.service.ts]: rename to something more descriptive, add a response type too

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "2.3.0",
   "license": "Apache License 2.0",
   "config": {
-    "webservice_version": "1.7.0-beta.1"
+    "webservice_version": "1.7.0-beta.2"
```
Comment (garyluu) [inline package.json]: RIP

Comment (coverbeck) [reply package.json]: I'm guessing this is why my tests are failing. :(

File: src/app/shared/refresh.service.ts (hunk 1 above)
Comment (coverbeck) [reply src/app/shared/refresh.service.ts]: Pushed fix.

File: src/app/shared/refresh.service.ts [hunk 3]
```diff
@@ -139,4 +139,28 @@ export class RefreshService {
...
     this.containerService.setTools(this.tools);
   }
+
+  syncServices(): void {
+    const message = 'Syncing services';
+    this.alertService.start(message);
+    this.usersService.syncUserServices().subscribe(
+      response => {
+        this.alertService.detailedSuccess();
+        this.workflowService.setWorkflows(response);
```
Comment (garyluu) [inline src/app/shared/refresh.service.ts]: Rename response to something more descriptive and add type.

**Comment by garyluu** [comment]
If you want, you can make an update workflows function that takes in Observable workflow and error message since the two sync functions (as well as others) are surprisingly similar

File: src/app/shared/refresh.service.ts (hunk 3 above)
Comment (coverbeck) [reply src/app/shared/refresh.service.ts]: Fixed by making a common function as you suggested below.

**Pull request merged**